.travis.yml

```diff
@@ -136,6 +136,14 @@ jobs:
         - mkdir -p ./deploy
         - aws s3 sync s3://$TRAVIS_S3_BUCKET/sdk-java/$TRAVIS_BUILD_NUMBER ./deploy
 
+      before_deploy:
+        - |
+          if [[ $TRAVIS_BRANCH =~ ^.?-dev$ ]]; then
+            sudo mv ./deploy/kuzzlesdk-*-$ARCH-javadoc.jar ./deploy/kuzzlesdk-$ARCH-experimental-javadoc.jar;
```

Comment:
The target should be `./deploy/kuzzlesdk-experimental-$ARCH-javadoc.jar`

Same goes for the other two targets below.